ioctl/newcmd/account/accountupdate_test.go

```diff
@@ -90,13 +90,12 @@ func TestNewAccountUpdate_FindPemFile(t *testing.T) {
 	accAddr, err := address.FromBytes(acc.Address.Bytes())
 	require.NoError(err)
 
-	oldWallet := config.ReadConfig.Wallet
-	config.ReadConfig.Wallet = testAccountFolder
-	defer func() {
-		config.ReadConfig.Wallet = oldWallet
-	}()
-
-	skPemPath := sm2KeyPath(accAddr)
+	client.EXPECT().Config().DoAndReturn(
+		func() config.Config {
+			config.ReadConfig.Wallet = testAccountFolder
+			return config.ReadConfig
```

Comment:
`config.ReadConfig` should be replaced with something like `TestConfig`

Reply:
yes, after modify all, i will create a pr to move config.ReadConfig into client.